### PR TITLE
Update instructions and test case description

### LINKED ISSUE #1818: Fruit Picker: Improve story/instructions
Some tasks in the fruit picker exercise are currently rather "technical" instead of being phrased in terms of the overall story. Maybe someone has a good idea who to improve this.

https://github.com/exercism/javascript/blob/main/exercises/concept/fruit-picker/.docs/instructions.md

General documentation: https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-instructions-md

### PR DESCRIPTION
**Pull request by alexmitchelldev**
Resolves #1818

Updated instructions.md to be less 'technical' and feel more part of the story. I also updated the test case description for the orderFromGrocer function to properly reflect the function name and arguments passed.

### CODE REVIEW

**Comment by SleeplessByte** [comment]
(Have not forgotten about this! I'll try to go through this this week <3)

**Comment by alexmitchelldev** [comment]
@SleeplessByte did you ever get chance to take a look at this?

**Comment by SleeplessByte** [comment]
@alexmitchelldev apologies. Short explanation at the end of this post https://forum.exercism.org/t/issue-with-local-jest-type-definitions-yarn-3-and-jest/6867/24

@ErikSchierboom @junedev @tejasbubane any of you time to check this?

File: exercises/concept/fruit-picker/.docs/instructions.md [hunk 1]
```diff
@@ -42,11 +44,11 @@ orderFromGrocer(
 // => `order` was called with the query and the callbacks
 ```
 
-## 4. Create a convenient short function
+## 4. Create a convenient helper function to place orders
```
Comment (ErikSchierboom) [inline exercises/concept/fruit-picker/.docs/instructions.md:47]: Maybe storify this a bit too? For example:

```suggestion
## 4. Simplify handling placed orders
```

or something like that

File: exercises/concept/fruit-picker/.docs/instructions.md [hunk 2]
```diff
@@ -42,11 +44,11 @@ orderFromGrocer(
...
 
-You find that you are calling this function from many different places with the same functions. Seeing an opportunity to refactor your code, you want to create a function where you can supply the variety and quantity to order as arguments.
+Your customers are now able to place fruit orders via your portal, however, you notice that you are invoking the `orderFromGrocer` function in many different places across your codebase, each time having to pass in a `query` and the 2 `callback` functions as arguments. Seeing an opportunity to refactor your code, you think it would be simpler if you could place an order by just passing the `variety` and `quanity` of fruit required. Define the `postOrder` helper function that takes `variety` and `quantity` as arguments and attempts to place an order with the grocer.
```
Comment (ErikSchierboom) [inline exercises/concept/fruit-picker/.docs/instructions.md:49]:
```suggestion
Your customers are now able to place fruit orders via your portal, however, you notice that you are invoking the `orderFromGrocer` function in many different places across your codebase, each time having to pass in a `query` and the two `callback` functions as arguments. Seeing an opportunity to refactor your code, you think it would be simpler if you could place an order by just passing the `variety` and `quantity` of fruit required. Define the `postOrder` helper function that takes `variety` and `quantity` as arguments and attempts to place an order with the grocer.
```

File: exercises/concept/fruit-picker/.docs/instructions.md [hunk 3]
```diff
@@ -2,27 +2,29 @@
 
 You are creating a new online portal for your patrons to order their fruit fresh from the grocer. The grocer has an API that you can use to see if they have the inventory desired by your customers. You need to create a small library of functions for interacting with the grocer's API.
 
-## 1. Create a callback to be called when the order is successful
+## 1. Let your customer know that their order was successful
```
Comment (ErikSchierboom) [inline exercises/concept/fruit-picker/.docs/instructions.md:5]: Maybe rephrase slightly to stress that the order might not be successful

```suggestion
## 1. Notify your customer when their order was successful
```

File: exercises/concept/fruit-picker/.docs/instructions.md [hunk 4]
```diff
@@ -2,27 +2,29 @@
...
-Write a callback function called `onSuccess` to be called when the order is successful. It should invoke the imported `notify` function passing a success message to it.
+The portal should notify your customer if their order was successful. Define the `onSuccess` callback function that will be called if the order was successful because the grocer _has_ the fruit in stock. This function should invoke the imported `notify` function, passing `{ message: 'SUCCESS' }` to let your customer know that their order was successful.
 
 ```javascript
 onSuccess();
 // => `notify` called with `{ message: 'SUCCESS' }`
 ```
 
-## 2. Create a callback to be called when the order fails with an error
+## 2. Let your customer know that their order was unsuccessful
```
Comment (ErikSchierboom) [inline exercises/concept/fruit-picker/.docs/instructions.md:14]:
```suggestion
## 2. Notify your customer when their order was unsuccessful
```

File: exercises/concept/fruit-picker/.docs/instructions.md [hunk 5]
```diff
@@ -2,27 +2,29 @@
 
 You are creating a new online portal for your patrons to order their fruit fresh from the grocer. The grocer has an API that you can use to see if they have the inventory desired by your customers. You need to create a small library of functions for interacting with the grocer's API.
 
-## 1. Create a callback to be called when the order is successful
+## 1. Let your customer know that their order was successful
 
-Write a callback function called `onSuccess` to be called when the order is successful. It should invoke the imported `notify` function passing a success message to it.
+The portal should notify your customer if their order was successful. Define the `onSuccess` callback function that will be called if the order was successful because the grocer _has_ the fruit in stock. This function should invoke the imported `notify` function, passing `{ message: 'SUCCESS' }` to let your customer know that their order was successful.
```
Comment (ErikSchierboom) [inline exercises/concept/fruit-picker/.docs/instructions.md:7]:
```suggestion
The portal should notify your customer if their order was successful. Define the `onSuccess` callback function that will be called if the order was successful due to the grocer having enough fruit in stock. This function should invoke the imported `notify` function, passing `{ message: 'SUCCESS' }` to let your customer know that their order was successful.
```

**Review by ErikSchierboom — COMMENTED** [review]
Could of small suggestions.

**Comment by alexmitchelldev** [comment]
@ErikSchierboom sorry about the glacial response time. I have committed your suggestions.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Thanks!
